server/graphql-schema.js

```diff
@@ -1016,6 +1016,7 @@ const graphqlSchema = gql`
         testPlanVersionId: ID!
         atId: ID!
         browserId: ID!
+        testPlanVersionDataToIncludeId: ID
```

Comment:
The part that confuses me is that it reads like we're copying data from a TestPlanVersion to a TestPlanReport, which superficially doesn't make sense, even if there's an algorithm happening under the hood. I find myself wondering if it would be possible to call this something like `copyResultsFromTestPlanReportId`. That seems more direct and intuitive to me. Do you think that would be workable?

Reply:
Fine with me! Done in https://github.com/w3c/aria-at-app/pull/978/commits/bffd2247824ed3d639b6576f8aad11085d539030.